activity-coordination.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Coordinating an Activity Area"
+author: "Graeme Stewart"
+layout: default
+---
+
+## Being an Activity Convener
+
+The HSF is extremely grateful to members of the community who take on the role
+of convener in our activity areas - *thank you*. Each group engages
+with a different subset of the developer and physics community and thus the
+exact pattern of activities that are organised can vary. It is delegated to the
+conveners to decide the way to organise so that it best serves the needs in
+HEP software and computing as required.
+
+### Meetings
+
+#### Activity Topical Events
+
+That said, organising semi-regular events is very important to keep up momentum
+in a topic and sometimes a series of related events might happen bunched
+together more tightly, e.g., looking at one specific topic from the point of
+view of several experiments.
+
+HSF activity meetings in general should try to:
+
+- be there to help encourage the exchange of ideas and techniques
+- do not need to focus on "finished" items, but rather can be there
+  to provide a community forum for unsolved problems
+- should ensure there is plenty of time for discussion
+- should take notes or minutes for important discussion points (attaching these
+  to Indico is recommended)
+
+#### HSF Seminars and Cross Interest Meetings
+
+Another route for a group is that instead of group specific meetings, organise
+contributors to the [HSF Seminar Series]({{ site.baseurl
+}}/meetings/hsf-seminars.html). This meeting has a regular cadence and would
```

Comment:
```suggestion
}}/meetings/hsf-seminars.html). These meetings have a regular cadence and would
```

Reply:
This I leave up to the native speaker

Reply:
Native speaker says yes, it's better as a plural!